znai/src/main/java/com/twosigma/znai/extensions/table/CsvParser.java

```diff
@@ -63,7 +63,7 @@ private static MarkupTableData parse(String content, List<String> header) {
 
             for (CSVRecord record : csvRecords) {
                 Row row = new Row();
-                record.forEach(v -> row.add(convert(v)));
+                record.forEach(row::add);
```

Comment:
Can't remember all the places, but it worth looking if we rely on numbers presence. 
One place could be a `:include-table:` plugin, it may automatically right align numbers.

Reply:
I see that auto alignment is *left* by default, not by a type of an object.


GfmTableToTableConverter.java

    private String extractAlignment(TableCell headerCell) {
        if (headerCell == null || headerCell.getAlignment() == null) {
            return "left";
        }

        switch (headerCell.getAlignment()) {
            case CENTER:
                return "center";
            case RIGHT:
                return "right";
            default:
                return "left";
        }
    }

Reply:
What about `:include-table: <jsonfile>` one?

Reply:
The change is only related to csv record, not json one.

Reply:
`:include-table:` works with both json and csv. Please double check if it relies on the fact that a certain value is a number when preparing the rendering info.

Reply:
I can't find any place to check a type of BigDecimal or Number from java files.

    public Column(String title, String align)  is the one accept align.

MarkupTableData.java is the only place to call Column(title, align) in addColumn().

    public void addColumn(String name, String align) {
        header.add(new Column(name, align));
    }

GfmTableToTableConverter.java is the only place to call MarkupTableData.addColumn().
    private void handleHeaderCell(TableCell headerCell) {
        tableData.addColumn(extractTitle(headerCell), extractAlignment(headerCell));
    }

Then I put extractAlignment() in the above comment.